src/wing/TiglWingStructureReference.h

```diff
@@ -0,0 +1,99 @@
+/* 
+* Copyright (C) 2014 Airbus Defense and Space
+*
+* Created: 2014-06-13 Roland Landertshamer
+*
+* Changed: $Id $ 
+*
+* Version: $Revision $
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#ifndef TIGLWINGSTRUCTUREREFERENCE_H
+#define TIGLWINGSTRUCTUREREFERENCE_H
+
+#include "tigl_internal.h"
+
+#include "PNamedShape.h"
+#include "TopoDS_Shape.hxx"
+#include "gp_Pnt.hxx"
+#include "gp_Vec.hxx"
+#include "TopoDS_Wire.hxx"
+#include "TopoDS_Face.hxx"
+
+namespace tigl
+{
+
+class CCPACSWing;
+class CCPACSWingSparSegment;
+class CCPACSWingRibsDefinition;
+class CCPACSWingCSStructure;
+//class CCPACSTrailingEdgeDevice;
+class CCPACSWingComponentSegment;
+
+// this is a set type, encapsulating a reference to either a CCPACSTrailingEdgeDevice or a CCPACSWingComponentSegment
+class TiglWingStructureReference
```

Comment:
Change class name and file name to CTiglWingStructureReference to match naming scheme of all other classes.